instrumentation/jvm-metrics/src/test/java/com/splunk/opentelemetry/jvmmetrics/JvmMetricsTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.jvmmetrics;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.splunk.opentelemetry.testing.TestMetricsAccess;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(AgentInstrumentationExtension.class)
+class JvmMetricsTest {
+  @Test
+  void shouldRegisterJvmMeters() {
+    var meterNames = TestMetricsAccess.getMeterNames();
+
+    // classloader metrics
+    assertTrue(meterNames.contains("jvm.classes.loaded"));
```

Comment:
Looking at `OtelNamingConvention`, I would expect seeing `runtime` somewhere here. Why not?

Reply:
Because `OtelNamingConvention` is not actually present in the test javaagent, we don't include `project(":custom")` there. In another PR I'd like to add metrics verification to smoke tests (once I figure out how...) so the full name will be checked there.